package.json

```
@@ -9,7 +9,7 @@
     "test": "nyc --check-coverage --lines 100 --functions 100 --branches 100 mocha app/tests",
     "coverage": "nyc report --reporter=text-lcov | node ./node_modules/coveralls/bin/coveralls.js",
     "lint": "./node_modules/.bin/eslint .",
-    "serve": "node ./node_modules/http-server/bin/http-server"
+    "serve": "npx serve"
```

```suggestion
    "serve": "npx serve -n"
```
Avoid ERROR: Cannot copy server address to clipboard: Both xsel and fallback failed.

---

Ah good call 👍